app/templates/components/table-of-contents.hbs

```diff
@@ -1,24 +1,29 @@
-{{#cp-panels accordion=true as |panels|}}
-  {{#each sortedGroupedResults as |result|}}
-    {{#panels.panel as |panel|}}
-      {{#panel.toggle}}
-        <li class="level-1">
-          <a class="slide-widget-head" href="#toc toc_deprecations-added-in-ember-2-x">
-            Deprecations added in {{result.since}}
-          </a>
-        </li>
-      {{/panel.toggle}}
-      {{#panel.body}}
-        <ol style="display:block">
-          {{#each result.contents as |content|}}
-            <li class="level-3">
-              <a href="#{{id-for-deprecation content.id}}">
-                {{content.title}}
-              </a>
-            </li>
-          {{/each}}
-        </ol>
-      {{/panel.body}}
-    {{/panels.panel}}
-  {{/each}}
-{{/cp-panels}}
\ No newline at end of file
+<div class="padding-vertical-small bg-light-muted rounded-sm">
```

Comment:
I'm not super sure what's going on when this component is rendered, but it's wrapping each `<li>` element in an `<a>` which is giving us a nested interactive element violation.